File: Week 1 - C/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

void print_row(int spaces, int bricks);

int main(void)
{
    int height;

    do
    {
        height = get_int("Height: ");
    }
    while (height < 1);

    for (int x = 0; x < height; x++)
    {
        print_row(height - x - 1, x + 1);
    }
}

void print_row(int spaces, int bricks)
{
    for (int y = 0; y < spaces; y++)
    {
        printf(" ");
    }

    for (int z = 0; z < bricks; z++)
    {
        printf("#");
    }
    printf("\n");
}

File: Week 6 - Python/sentimental-mario-less/mario.py
import cs50

height = 0
while height < 1 or height > 8:
    height = cs50.get_int("Height: ")

x = 1
for i in range(height):
    for _ in range(height - x):
        print(" ", end="")
    for j in range(x):
        print("#", end="")
    x += 1
    print("")
